pkg/rclone/rcserver/rc.go

```diff
@@ -616,6 +660,20 @@ func init() {
 - dstFs - a remote name string eg "drive2:" for the destination
 - dstRemote - a directory path within that remote for the destination`,
 	})
+
+	rc.Add(rc.Call{
+		Path:         "sync/copypaths",
+		AuthRequired: true,
+		Fn:           wrap(rcCopyPaths(), remoteToLocal()),
+		Title:        "Copy paths from source directory to destination",
+		Help: `This takes the following parameters:
+
+- srcFs - a remote name string eg "drive:" for the source
```

Comment:
From my pov `s3` is more informative than drive.

> - fs - a remote name string eg "`s3`:path/to/file"

Some other descriptions already have `s3` instead of `drive` in them, so I will `drive` to `s3` in the whole file.